src/server/pfs/db/driver.go

```diff
@@ -1165,12 +1167,9 @@ func getPrefixes(path string) []string {
 }
 
 func getDiffID(commitID string, path string) string {
-	return fmt.Sprintf("%s:%s", commitID, path)
-}
-
-// the equivalent of above except that commitID is a rethink term
-func getDiffIDFromTerm(commitID gorethink.Term, path string) gorethink.Term {
-	return commitID.Add(":" + path)
+	s := fmt.Sprintf("%s:%s", commitID, path)
+	hash := sha256.Sum256([]byte(s))
```

Comment:
This would be pretty easy to change, go has a bunch of hash functions and they all implement the same interface. Looking at them though it seems like they're all smaller output sizes, 64 bit being the largest so that would increase the chances of collision. Also sha is what git uses, for much the same purpose we're using it for here, so I think it's a reasonable choice.

Interesting SO post about this: http://stackoverflow.com/questions/28792784/why-does-git-use-a-cryptographic-hash-function